mpp-ios.xcodeproj/xcuserdata/kencho.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>mpp-ios.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
```

Comment:
Should `orderHint` be more descriptive?

Reply:
This is user settings, so we'll put it in gitignore now, if that's okay!

Reply:
Sounds good!